### PR TITLE
Topic not found

### PR DESCRIPTION
**Pull request by CJPoll**
Handle the case where a topic isn't found. Also increase debug logging.

### CODE REVIEW

File: lib/rivulet/avro/registry.ex [hunk 1]
```diff
@@ -29,14 +37,18 @@ defmodule Rivulet.Avro.Registry do
...
   end
 
   @spec get_schema(Avro.schema_id) :: {:ok, Schema.t} | {:error, term}
   def get_schema(schema_id) do
+    Logger.debug("Attempting to get schema: #{schema_id}")
     case get("/schemas/ids/#{schema_id}") do
       {:ok, resp} -> handle_get_schema_response(resp, schema_id)
-      {:error, _} = err -> err
+      {:error, reason} = err ->
+        Logger.debug("Could not get schema: #{schema_id}. Probably couldn't connect to registry. Reason: #{inspect reason}")
```
Comment (aweagel) [inline lib/rivulet/avro/registry.ex]: Should this one be `Logger.error` as well?

File: lib/rivulet/kafka/stage/publisher.ex [hunk 2]
```diff
@@ -72,6 +72,19 @@ defmodule Rivulet.Kafka.Stage.Publisher do
   @spec handle_event(Message.t) :: ignored
   def handle_event(%Message{} = msg) do
     Logger.debug("Publishing #{inspect msg}")
-    Publisher.publish(msg.topic, msg.partition_strategy, msg.encoding_strategy, msg.key, msg.value)
+    case Publisher.publish(msg.topic, msg.partition_strategy, msg.encoding_strategy, msg.key, msg.value) do
+      nil ->
+        Logger.debug("Publish returned nil?")
```
Comment (aweagel) [inline lib/rivulet/kafka/stage/publisher.ex]: Is the `?` a typo?

Comment (CJPoll) [reply lib/rivulet/kafka/stage/publisher.ex]: Done

File: lib/rivulet/avro/registry.ex (hunk 1 above)
Comment (CJPoll) [reply lib/rivulet/avro/registry.ex]: Done

**Pull request merged**